### PR TITLE
Remove some logging messages

### PR DESCRIPTION
**Pull request by sduzh**

### CODE REVIEW

**Review by imay — DISMISSED** [review]

**Review by chaoyli — DISMISSED** [review]

File: be/src/storage/vectorized/delta_writer.cpp [hunk 1]
```diff
@@ -281,7 +281,6 @@ Status DeltaWriter::close_wait(google::protobuf::RepeatedPtrField<PTabletInfo>*
     _delta_written_success = true;
 
     const FlushStatistic& stat = _flush_token->get_stats();
-    LOG(INFO) << "Closed delta writer. tablet_id=" << _tablet->tablet_id() << " stats=" << stat;
```
Comment (wyb) [inline be/src/storage/vectorized/delta_writer.cpp:284]: Flush time, count, size are usefull, better not to remove.

Comment (sduzh) [reply be/src/storage/vectorized/delta_writer.cpp]: > Flush time, count, size are usefull, better not to remove.

updated, PTAL

**Comment by sduzh** [comment]
run starrocks_tscancode

**Comment by sduzh** [comment]
run starrocks_tscancode

**Review by chaoyli — APPROVED** [review]

**Review by imay — APPROVED** [review]

**Pull request merged**